PollBuddy-Server/frontend/src/pages/GroupJoin/GroupJoin.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import "mdbreact/dist/css/mdb.css";
+import { MDBContainer } from "mdbreact";
+
+export default class GroupJoin extends Component {//this class will likely need to call Groups/new and do more with that...
+  componentDidMount(){
+    this.props.updateTitle("Group Join");
```

Comment:
This is probably silly but "Join Group" sounds a lot better to me than "Group Join".

Reply:
I'll just change it quickly so you don't have to bother with it.